README.md

```diff
@@ -212,3 +212,6 @@ After this, the code directory should look like C:\php-sdk\phpdev\vc15\x64\php-s
 
 ## Troubleshooting
 If you encounter problem like how to ensure redirection or PHP crash when using this extension, you may follow the steps described in [troubleshooting.md](/troubleshooting.md) for initial troubleshooting.
```

Comment:
Reword the line to  "To troubleshoot issues when using this extension, you may follow the steps described in [troubleshooting.md](/troubleshooting.md) for initial troubleshooting. " 

No need to call out PHP crash or redirection issues specifically its implicit here

Reply:
+1
Calling out 'crash' word probably confuse Cx we may have potential bug

Reply:
updated